tests/playground.html

```diff
@@ -67,6 +67,9 @@
 <script src="blocks/test_blocks.js"></script>
 <script src="themes/test_themes.js"></script>
 <script src="./playgrounds/screenshot.js"></script>
+<script src="./plugins/toolbox_new.js"></script>
```

Comment:
Maybe this should live under playgrounds.

Reply:
They can, I'm not sure I was planning on even including these in final pr, just seemed useful for discussion purposes.

Reply:
Totally, then please ignore.